tests/application.py

```diff
@@ -11,4 +21,5 @@ def test_style():
 
     style = pycodestyle.StyleGuide(ignore=['E501', 'W504'])
     result = style.check_files(['config/', 'v1/'])
+    print(result.total_errors)
```

Comment:
@vosi is that left from debugging? If not, maybe add more verbose print, like 'total number of errors: ...;